src/components/IncidentsFilterBar/index.tsx

```diff
@@ -56,7 +56,7 @@ export default function TopologyOpenIncidentsFilterBar({
         name="type"
         control={control}
         prefix={"Type:"}
-        className="border-none shadow-none font-semibold space-x-3 capitalize text-base px-2"
+        className="border-none shadow-none font-semibold space-x-3 capitalize text-sm px-2"
```

Comment:
I think these classes should be the defaults of the dropdown - we shouldn't be styling individual dropdowns